### DISCUSSION TITLE
Love this project - what is the path to production?

### DISCUSSION THREAD

**Discussion by shihzy** [imartinez/privateGPT#637] [General]
Hi all - love this project and the simplicity to creating a private LLM. Especially for folks like myself who are tinkering but are not that deep technically (I'm just an ole' PM)

With that - I'm curious what the path to production might look like? It seems like the user persona here is really a semi-technical engineer who has expertise in other areas (i.e. front end) but doesn't want to deal with the LLM side of it. And also needs to privacy piece as a core tenant of the product.

As someone just running it locally on a Mac (and not terribly technical), its def a bit slow. Some of the responses are a bit "interesting".

Curious what others in the community think.

**Comment by sime2408** [comment]
You need a cloud environment with a GPU instance which is very costly in the cloud, that's why a lot of folks tend to run their own machine somewhere on the edge, but if you need backbone infrastructure you need a cloud. For the project itself, you would probably dockerize it and put Chroma DB accessible over an IP address to store its data in some database, Hadoop, or similar. If you want concurrency (multiple users logged in this project is not ready for that. I guess you need login with Google or something with a database of users creating sessions for them

**Reply by yash-tractebel-LLM** [reply]
Hello, does anyone know if there is a concurrency possibility yet (multiple users able to use over the enterprise cloud?)

Thanks.

**Reply by JBGitHub11** [reply]
Interested in this as well.
